docs/admin/admission-controllers.md

```diff
@@ -440,47 +482,68 @@ The priority admission controller uses the `priorityClassName` field and populat
 
-It is strongly encouraged that this plug-in is configured last in the sequence of admission control plug-ins.  This is
+It is strongly encouraged that this admission controller is configured last in the sequence of admission controllers.  This is
 so that quota is not prematurely incremented only for the request to be rejected later in admission control.
 
 ### SecurityContextDeny
 
-This plug-in will deny any pod that attempts to set certain escalating [SecurityContext](/docs/user-guide/security-context) fields. This should be enabled if a cluster doesn't utilize [pod security policies](/docs/user-guide/pod-security-policy) to restrict the set of values a security context can take.
+This admission controller will deny any pod that attempts to set certain escalating [SecurityContext](/docs/user-guide/security-context) fields. This should be enabled if a cluster doesn't utilize [pod security policies](/docs/user-guide/pod-security-policy) to restrict the set of values a security context can take.
 
 ### ServiceAccount
 
-This plug-in implements automation for [serviceAccounts](/docs/user-guide/service-accounts).
-We strongly recommend using this plug-in if you intend to make use of Kubernetes `ServiceAccount` objects.
+This admission controller implements automation for [serviceAccounts](/docs/user-guide/service-accounts).
+We strongly recommend using this admission controller if you intend to make use of Kubernetes `ServiceAccount` objects.
+
+### ValidatingAdmissionWebhook (alpha in 1.8; beta in 1.9)
+
+This admission controller calls any validating webhooks which match the request. Matching
+webhooks are called in parallel; if any of them rejects the request, the request
+fails. This admission controller only runs in the validation phase; the webhooks it calls may not
+mutate the object, as opposed to the webhooks called by the `MutatingAdmissionWebhook` admission controller.
+
+If a webhook called by this has side effects (for example, decrementing quota) it
+*must* have a reconcilation system, as it is not guaranteed that subsequent
+webhooks or other validating admission controllers will permit the request to finish.
 
+If you disable the ValidatingAdmissionWebhook, you must also disable the
+`ValidatingWebhookConfiguration` object in the `admissionregistration/v1beta1`
+group/version via the `--runtime-config` flag (both are on by default in
+versions >= 1.9).
 
-## Is there a recommended set of plug-ins to use?
+
+## Is there a recommended set of admission controllers to use?
 
 Yes.
-For Kubernetes >= 1.6.0, we strongly recommend running the following set of admission control plug-ins (order matters):
+For Kubernetes >= 1.9.0, we strongly recommend running the following set of admission controllers (order matters):
+
+```shell
+--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ValidatingAdmissionWebhook,ResourceQuota,DefaultTolerationSeconds,MutatingAdmissionWebhook
```

Comment:
Leaving `MutatingAdmissionWebhook` at last might be confusing.

Reply:
Added more explanation instead. It's more important to explain how the flag works IMO.